code/modules/mob/living/silicon/robot/laws.dm

```diff
@@ -28,6 +29,7 @@
 
 	who << "<b>Obey these laws:</b>"
 	laws.show_laws(who)
+	// TODO: Update to new antagonist system.

```

Comment:
While the new antag system is indeed more pleasant to work with than the old variant, it still has some issues, most notably:
* Certain game modes don't end properly, Heist specifically.
* Automatic mid-round antags doesn't currently appear to work.
* Due to the point above Secret is always Extended.
You probably wish for these issues to be fixed first.